checkstyle/suppressions.xml

```diff
@@ -25,7 +25,7 @@
               files="(AbstractKafkaAvroDeserializer|CompatibilityResource|KafkaSchemaRegistry|KafkaStoreReaderThread|AvroData|DownloadSchemaRegistryMojo).java"/>
 
     <suppress checks="NPathComplexity"
-              files="(AvroData|DownloadSchemaRegistryMojo).java"/>
+              files="(AvroData|DownloadSchemaRegistryMojo|CompatibilityResource).java"/>
```

Comment:
@stanislavkozlovski Can you check if you have this commit: https://github.com/confluentinc/schema-registry/commit/a0ec756252f6bc676098824705d6f451f075dd38 ? I could run the latest build without this suppression.